google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractConnectionImplTest.java

```diff
@@ -77,9 +89,24 @@
   private static final String LOG_FILE =
       "src/test/resources/com/google/cloud/spanner/connection/ConnectionImplGeneratedSqlScriptTest.sql";
 
+  private static final String PG_LOG_FILE =
+      "src/test/resources/com/google/cloud/spanner/connection/PG_ConnectionImplGeneratedSqlScriptTest.sql";
```

Comment:
nit: would it be better to have a googlesql folder / pg folder? (`src/test/resources/com/google/cloud/spanner/connection/postgresql/ConnectionImplGeneratedSqlScriptTest.sql`)

Reply:
I've added a specific PG folder. Many of the files are also shared, and I don't feel it would make sense to add those to one of the folders, so I've kept all default + Google SQL files in the main directory, and only moved the PG specific ones to the new folder.